### DISCUSSION TITLE
Preview Deployment Comments on Alias Domains

### DISCUSSION THREAD

**Discussion by Merott** [vercel/community#1326] [Help] [answered]
### Summary

**The key question:**

Are Preview Deployment Comments not available on manually assigned alias domains?

**Full context:**

I have a Next.js project deploying to Vercel with GitHub Actions. It's not using automatic deployments with GitHub integration because I want more control over the pipeline.

I'm also using the Vercel CLI (`vercel alias set`) to assign alias domains to my Preview deployments after the deployment is complete.

However, I've noticed the commenting functionality on Vercel previews is only available with the default domains assigned by Vercel itself, not with my custom alias domains. For example:

https://polarhabits-cwp6g1izh-polarpixels.vercel.app (automatically assigned -- commenting options present)
https://polarhabits--ph-45-datadog.vercel.app (custom alias -- commenting options not present)

Those two are the exact same deployment.

I coudln't find any documentation that says commenting is only available on automatically assigned aliases, so I'm wondering if this is expected behaviour, or perhaps a bug?

I can see `feedback.js` is loading on both sites, which leads me to believe this is a bug…

**Comment by franknoel** [comment]
I have the same bug for my sites with Next.js 13.1.6.

**Comment by amyegan** [comment, marked as answer]
When [using comments with a custom alias domain](https://vercel.com/docs/concepts/deployments/comments/specialized-usage#using-comments-with-a-custom-alias-domain), you need to explicitly [opt into comments in your project settings](https://vercel.com/docs/concepts/deployments/comments/how-comments-work#at-the-project-level).

**Reply by franknoel** [reply]
Thank you!
